src/scene/mesh/shared/MeshPipe.ts

```diff
@@ -215,7 +217,7 @@ export class MeshPipe implements RenderPipe<Mesh>, InstructionPipe<Mesh>
         // TODO - make this batchable graphics??
         const gpuMesh: BatchableMesh = BigPool.get(BatchableMesh);
 
-        gpuMesh.mesh = mesh;
```

Comment:
`mesh` is assigned a couple of lines below. I assume it is supposed to be a `geometry` assignment here. But the missing `geometry` assignement didn't cause a bug, because we currently always update the batchable mesh in `addRenderable`/`updateRenderable` (there's no `_didMeshUpdate` check like with Sprite).